tox.ini

```diff
@@ -6,3 +6,13 @@ deps = pytest
 install_command = pip install -e .[test] --find-links {toxinidir}/3rdparty {opts} {packages}
 # TODO(xujyan): Fix up 'test_scheduler_runs'.
 commands = py.test -v -k 'not test_scheduler_runs' tests {posargs:}
+
+[testenv:style]
+deps =
+  twitter.common.app==0.3.0
+  twitter.common.collections==0.3.0
+  twitter.common.lang==0.3.0
+  twitter.common.log==0.3.0
+  twitter.checkstyle==0.1.0
```

Comment:
When I didn't have them tox told me these dependencies were required but not installed...
However the PEX example runs fine on my machine.

One possibility is that in the PEX tox.ini the Twitter dependencies were specified as ">=0.3.1,<0.4.0", which I tried but didn't work for Mysos because if I have a custom Twitter package that depends on the public Mysos package and the Twitter package also has certain t.c.* dependencies then ">=0.3.1,<0.4.0" would cause version conflict.

Oh I can see where the headache begins but so far I was able to get away with pining t.c.* packages to a specific version.

twitter.checkstyle depends on the 0.3.0 version of t.c.* which is different from the version I use for Mysos itself.